scripts/lstchain_data_r0_to_dl1.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from ctapipe.utils import get_dataset_path
+from lstchain.reco import dl0_to_dl1
+from lstchain.io.config import read_configuration_file
+import os
+
+parser = argparse.ArgumentParser(description="R0 to DL1")
+
+
+parser.add_argument('--infile', '-f', type=str,
+                    dest='infile',
+                    help='path to the file with simtelarray events',
+                    default=get_dataset_path('gamma_test_large.simtel.gz'))
+
+parser.add_argument('--outdir', '-o', action='store', type=str,
+                    dest='outdir',
+                    help='Path where to store the reco dl2 events',
+                    default='./dl1_data/')
+
+parser.add_argument('--config_file', '-conf', action='store', type=str,
+                    dest='config_file',
+                    help='Path to a configuration file. If none is given, a standard configuration is applied',
+                    default=None
+                    )
+
+parser.add_argument('--pedestal_path', '-pedestal', action='store', type=str,
+                    dest='pedestal_path',
+                    help='Path to a pedestal file',
+                    default=None
+                    )
+
+parser.add_argument('--calibration_path', '-calib', action='store', type=str,
+                    dest='calibration_path',
+                    help='Path to a calibration file',
+                    default=None
+                    )
+
+
+args = parser.parse_args()
+
+
+if __name__ == '__main__':
+    os.makedirs(args.outdir, exist_ok=True)
+    
+    dl0_to_dl1.allowed_tels = {1, 2, 3, 4}
+    output_filename = args.outdir + '/dl1_' + os.path.basename(args.infile).rsplit('.', 1)[0] + '.h5'
```

Comment:
This will produce filenames as: `dl1_LST-1.1.Run01555.0000.fits.h5`. Maybe we should take out the fits part as well to have it like: `dl1_LST-1.1.Run01555.0000.h5`

Reply:
you are right, as right now it produces things like `***.simtel.h5`.
cons: that's a double file extension
pros: it keeps the info about origin
I don't have strong preferences.

Reply:
Leave it as it is then